src/libraries/System.Net.WebSockets.Client/src/System/Net/WebSockets/BrowserWebSockets/ClientWebSocketOptions.cs

```diff
@@ -32,6 +32,20 @@ public bool UseDefaultCredentials
             set => throw new PlatformNotSupportedException();
         }
 
+        [UnsupportedOSPlatform("browser")]
+        public Version Version
+        {
+            get => throw new PlatformNotSupportedException();
+            set => throw new PlatformNotSupportedException();
+        }
+
+        [UnsupportedOSPlatform("browser")]
+        public System.Net.Http.HttpVersionPolicy VersionPolicy
+        {
+            get => throw new PlatformNotSupportedException();
```

Comment:
Presumably the getters could be supported and just always return the defaults?

Reply:
> Should I add [UnsupportedOSPlatform("browser")] attribute in that case for setter only?

If we make the getter work everywhere, then yes, the unsupported attribute would move to just the setter.